Firestore/Source/Local/FSTMemoryQueryCache.h

```diff
@@ -16,6 +16,7 @@
 
 #import <Foundation/Foundation.h>
 
+#import "Firestore/Source/Local/FSTLocalSerializer.h"
```

Comment:
Conventionally in Objective-C we've been using forward declarations (with `@class`) to keep compilation times down.

Note that this works pretty well in Objective-C but basically doesn't work for C++ so we use the IWYU rule there.

Reply:
Ok, switched to a forward declaration, here and elsewhere.